### PR TITLE
Megan henry

### PR DESCRIPTION
**Pull request by COpenda**
Do not merge. To add images and change date.

### CODE REVIEW

File: _posts/2016-08-24-megan-henry.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+tags: [data]
+author_id: 13
+status: true
+type: post
+published: true
+---
+
+We are happy to introduce Megan Schroeder and Henry Ng’enda, the new members of the Ona team. 
+
+Megan is taking over from Sylvia Musula as our new support and M&E manager, handling client projects and overseeing the support team. She has a degree in biomedical engineering from the University of Minnesota, a Master's degree in Public Health from Emory University, and is fluent in spoken and written Swahili. Megan has an aptitude for leveraging technology for M&E and brings on board a wealth of experience in project development and evaluation. When she’s not delivering world class support to our users, Megan enjoys listening to bongo music or reading a good book.
```
Comment (ukanga) [inline _posts/2016-08-24-megan-henry.md]: Should we capitalize `Biomedical Engineering`?

Comment (ukanga) [inline _posts/2016-08-24-megan-henry.md]: Should we indicate she is fluent in English as well? Swahili is relevant as a fun fact but ths could indicate that she is only fluent in that language.

Comment (ukanga) [inline _posts/2016-08-24-megan-henry.md]: Probably we should rephrase this `Megan has an aptitude for leveraging technology for M&E and brings on board a wealth of experience in project development and evaluation.`

Something in the vein of `Megan has an aptitude for leveraging technology for M&E projects and brings on board a wealth of experience in project development and evaluation.`

File: _posts/2016-08-24-megan-henry.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Henry has joined us a software engineer for the Ona platform. He is extremely passionate about developing automated systems that address people's daily challenges. Henry has a bachelor's degree in commerce, and holds a master's degree in information systems from the University of Nairobi. He is currently working on adding new features and extending Ona’s backend. When he’s not building software for change, Henry enjoys reading about new tech advancements and spending time with his family.
+
+To greater heights!
```
Comment (ukanga) [inline _posts/2016-08-24-megan-henry.md]: Not sure about this statement.

File: _posts/2016-08-24-megan-henry.md (hunk 1 above)
Comment (COpenda) [inline _posts/2016-08-24-megan-henry.md]: @ukanga  According to @rowo a Master's degree is what's meant to be capitalized, not a bachelor's degree. 

Making the other corrections as well. Thanks for taking a look at this.

**Pull request merged**